### PR TITLE
Update Phragmms

### PR DESCRIPTION
**Pull request by a-jwc**
- Removed time complexities and simplified c-factor approximations
- Moved Phragmms to Optimizations section
- Update wording and clarified motivations for the new election rule

### CODE REVIEW

**Comment by DannyS03** [comment]
Merge is pending a build fix, but review can happen.

**Comment by AlfonsoCev** [comment]
I sent some some feedback to Alex via Element. Currently the last algorithm description of Phragmms is hard to understand.

**Review by salmad3 — APPROVED** [review]

File: docs/learn/learn-phragmen.md [hunk 1]
```diff
@@ -147,8 +110,8 @@ The Phragmén method will iterate, selecting one seat at a time, according to th
    modified after submission.
 2. An initial load of 0 is set for each ballot.
 3. The candidate who wins the next available seat is the one where the ballots of their supporters
-   would have the *least average (mean) cost* if that candidate wins.
-4. The *n* ballots that approved that winning candidate get *1/n* added to their load.
+   would have the _least average (mean) cost_ if that candidate wins.
```
Comment (DannyS03) [inline docs/learn/learn-phragmen.md:113]:
```suggestion
   would have the *least average (mean) cost* if that candidate wins.
```

File: docs/learn/learn-phragmen.md [hunk 2]
```diff
@@ -147,8 +110,8 @@ The Phragmén method will iterate, selecting one seat at a time, according to th
...
+4. The _n_ ballots that approved that winning candidate get _1/n_ added to their load.
```
Comment (DannyS03) [inline docs/learn/learn-phragmen.md:114]:
```suggestion
4. The *n* ballots that approved that winning candidate get *1/n* added to their load.
```

File: docs/learn/learn-phragmen.md [hunk 3]
```diff
@@ -270,8 +233,8 @@ Weighted Phragmén is similar to Basic Phragmén in that it selects candidates s
 round, until the maximum number of candidates are elected. However, it has additional features to
 also allocate weight (stake) behind the candidates.
 
-*Note: in terms of validator selection, for the following algorithm, you can think of "voters" as
-"nominators" and "candidates" as "validators".*
+_Note: in terms of validator selection, for the following algorithm, you can think of "voters" as
+"nominators" and "candidates" as "validators"._
```
Comment (DannyS03) [inline docs/learn/learn-phragmen.md:237]:
```suggestion
"nominators" and "candidates" as "validators".*
```

File: docs/learn/learn-phragmen.md [hunk 4]
```diff
@@ -283,8 +246,8 @@ In more depth, the algorithm operates like so:
 
 1. Create a list of all voters, their total amount of stake, and which validators they support.
 2. Generate an initial edge-weighted graph mapping from voters to candidates, where each edge weight
-   is the total *potential* weight (stake) given by that voter. The sum of all potential weight for
-   a given candidate is called their *approval stake*.
+   is the total _potential_ weight (stake) given by that voter. The sum of all potential weight for
```
Comment (DannyS03) [inline docs/learn/learn-phragmen.md:249]:
```suggestion
   is the total *potential*weight (stake) given by that voter. The sum of all potential weight for
```

File: docs/learn/learn-phragmen.md [hunk 5]
```diff
@@ -283,8 +246,8 @@ In more depth, the algorithm operates like so:
...
+   a given candidate is called their _approval stake_.
```
Comment (DannyS03) [inline docs/learn/learn-phragmen.md:250]:
```suggestion
   a given candidate is called their *approval stake*.
```

**Pull request merged**